Source/WebCore/svg/graphics/filters/SVGFilter.h

```diff
@@ -39,6 +39,7 @@ class SVGFilter final : public Filter {
     static RefPtr<SVGFilter> create(SVGFilterElement&, RenderingMode, const FloatSize& filterScale, ClipOperation, const FloatRect& filterRegion, const FloatRect& targetBoundingBox, const GraphicsContext& destinationContext);
     WEBCORE_EXPORT static RefPtr<SVGFilter> create(const FloatRect& targetBoundingBox, SVGUnitTypes::SVGUnitType primitiveUnits, SVGFilterExpression&&);
 
+    static bool isNoop(SVGFilterElement&);
```

Comment:
Maybe `isIdentity()` would be easier to read (replace all the "Noop" with "isIdentity").